### PR TITLE
update the test org instance url since the my domain is turned on

### PR DESCRIPTION
**Pull request by helenren**

### CODE REVIEW

**Comment by wmathurin** [comment]
👍

File: libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/ui/SalesforceHybridTestActivity.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 	static String refreshToken = "5Aep861KIwKdekr90IDidO4EhfJiYo3fzEvTvsEgM9sfDpGX0qFFeQzHG2mZeUH_.XNSBE0Iz38fnWsyYYkUgTz";
 	static String authToken = "--will-be-set-through-refresh--";
 	static String identityUrl = "https://test.salesforce.com";
-	static String instanceUrl = "https://cs1.salesforce.com";
+	static String instanceUrl = "https://mobilesdk.cs1.my.salesforce.com";
```
Comment (bhariharan) [inline libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/ui/SalesforceHybridTestActivity.java]: `instanceUrl` should be `cs1.salesforce.com`. `communityUrl` would be `mobilesdk.cs1.my.salesforce.com`.

File: libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/accounts/UserAccountTest.java [hunk 2]
```diff
@@ -47,7 +47,7 @@
     public static final String TEST_USERNAME = "test_username";
     public static final String TEST_CLIENT_ID = "test_client_d";
     public static final String TEST_LOGIN_URL = "https://test.salesforce.com";
-    public static final String TEST_INSTANCE_URL = "https://cs1.salesforce.com";
+    public static final String TEST_INSTANCE_URL = "https://mobilesdk.cs1.my.salesforce.com";
```
Comment (bhariharan) [inline libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/accounts/UserAccountTest.java]: Same comment as above.

File: libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/ClientManagerTest.java [hunk 3]
```diff
@@ -66,7 +66,7 @@
     public static final String TEST_USERNAME = "test_username";
     public static final String TEST_CLIENT_ID = "test_client_d";
     public static final String TEST_LOGIN_URL = "https://test.salesforce.com";
-    public static final String TEST_INSTANCE_URL = "https://cs1.salesforce.com";
+    public static final String TEST_INSTANCE_URL = "https://mobilesdk.cs1.my.salesforce.com";
```
Comment (bhariharan) [inline libs/test/SalesforceSDKTest/src/com/salesforce/androidsdk/rest/ClientManagerTest.java]: Same comment as above.

File: libs/test/SmartStoreTest/res/values/bootconfig.xml [hunk 4]
```diff
@@ -7,10 +7,10 @@
         <item>api</item>
     </string-array>
     <string name="androidPushNotificationClientId"></string>
-    <string name="account_name">ut2@cs1.mobilesdk.org (https://cs1.salesforce.com) (test)</string>
+    <string name="account_name">ut2@cs1.mobilesdk.org (https://mobilesdk.cs1.my.salesforce.com) (test)</string>
     <string name="account_type">com.salesforce.androidsdk.smartstoretest.login</string>
     <string name="encoding">UTF-8</string>
-    <string name="instance_url">https://cs1.salesforce.com</string>
+    <string name="instance_url">https://mobilesdk.cs1.my.salesforce.com</string>
```
Comment (bhariharan) [inline libs/test/SmartStoreTest/res/values/bootconfig.xml]: Same comment as above.

File: native/NativeSampleApps/test/RestExplorerTest/src/com/salesforce/samples/restexplorer/ExplorerActivityTest.java [hunk 5]
```diff
@@ -71,7 +71,7 @@
     private static final String TEST_USER_ID = "test_user_id";
     private static final String TEST_CLIENT_ID = "test_client_d";
     private static final String TEST_LOGIN_URL = "https://test.salesforce.com";
-    private static final String TEST_INSTANCE_URL = "https://cs1.salesforce.com";
+    private static final String TEST_INSTANCE_URL = "https://mobilesdk.cs1.my.salesforce.com";
```
Comment (bhariharan) [inline native/NativeSampleApps/test/RestExplorerTest/src/com/salesforce/samples/restexplorer/ExplorerActivityTest.java]: Same comment as above.

File: libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/ui/SalesforceHybridTestActivity.java (hunk 1 above)
Comment (helenren) [inline libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/ui/SalesforceHybridTestActivity.java]: For some tests required on ios side, we just turn the My.Dormain on using the customer domain mobilesdk for our test org.

Comment (helenren) [inline libs/test/SalesforceHybridTest/src/com/salesforce/androidsdk/phonegap/ui/SalesforceHybridTestActivity.java]: Figured out, you are right. Will update it. Thanks

**Pull request merged**